game/stronghold_crusader_aiv.ksy

```diff
@@ -0,0 +1,104 @@
+meta:
+  id: aiv
+  title: AI Village
+  application:
+    - Stronghold Crusader
+    - Stronghold Crusader Extreme
+    - Village
+  file-extension: aiv
+  license: CC0-1.0
+  endian: le
+doc: |
+  The structure of the Stronghold Crusader AI Village file format is similar to
+  the one of the map file format. It consists of a directory, specifying
+  properties of the following data sections. Some sections are compressed using
+  blast compression, an implementation of which can be found at
+  https://github.com/ladislav-zezula/StormLib/tree/master/src/pklib.
+seq:
+  - id: dir
+    type: dir
+  - id: x_view
+    type: u4
+  - id: y_view
+    type: u4
+  - id: random
+    size: 40016
+  - id: bmap_size
+    type: compr_sec
+  - id: bmap_tile
+    type: compr_sec
+  - id: tmap
+    type: compr_sec
+  - id: gmap
+    size: 10000
+  - id: bmap_id
+    type: compr_sec
+  - id: bmap_step
+    type: compr_sec
+  - id: step_cur
+    type: u4
+  - id: step_tot
+    type: u4
+  - id: parr
+    type: s4
+    repeat: expr
+    repeat-expr: _root.dir.uncompr_size[11]/4 # always 10 or 50
+  - id: tarr
+    type: u4
+    repeat: expr
+    repeat-expr: 240
+  - id: pause
+    type: u4
```

Comment:
Is this a time interval? If so, in what time units (maybe seconds, milliseconds, microseconds)?